### ISSUE TITLE
schema-reader Error: Undefined array key "order"

### ISSUE THREAD

**Issue by MeisamMulla** [VemtoOrg/vemto2-issues#57]
This happened after I created a parent child relationship for the same model.

**Migration:**
```php
    public function up(): void
    {
        Schema::table('roms', function (Blueprint $table) {
            $table->unsignedBigInteger('parent_id')->nullable()->after('id');
            $table->foreign('parent_id')->references('id')->on('roms')->onDelete('restrict');
        });
    }

    public function down(): void
    {
        Schema::table('roms', function (Blueprint $table) {
            $table->dropForeign(['parent_id']);
            $table->dropColumn('parent_id');
        });
    }
```

**Error Message:**
schema-reader Error: Undefined array key "order"

**Stack:**
```
schema-reader Error: #0 /Users/meisam/Code/roms-backend/vendor/laravel/framework/src/Illuminate/Foundation/Bootstrap/HandleExceptions.php(256): Illuminate\Foundation\Bootstrap\HandleExceptions->handleError(2, 'Undefined array...', 'phar:///Applica...', 69)
#1 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php(69): Illuminate\Foundation\Bootstrap\HandleExceptions->Illuminate\Foundation\Bootstrap\{closure}(2, 'Undefined array...', 'phar:///Applica...', 69)
#2 [internal function]: TableRepository->{closure}(Array)
#3 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php(68): array_map(Object(Closure), Array)
#4 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php(49): TableRepository->insertColumn('roms', Array)
#5 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php(36): TableRepository->addTableColumn(Array)
#6 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/classes/TableRepository.php(21): TableRepository->processAddedColumns(Array)
#7 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/index.php(82): TableRepository->buildTablesFromMigrations()
#8 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/common/Vemto.php(107): {closure}()
#9 phar:///Applications/Vemto.app/Contents/Resources/static/VMTTL1/index.php(18): Vemto::execute('schema-reader', Object(Closure))
#10 /Applications/Vemto.app/Contents/Resources/static/VMTTL1(15): require('phar:///Applica...')
#11 {main}
```

---

Vemto 2.0.5
darwin 23.0.0
Locale: en-US
App version: 2.0.5

**Comment by TiagoSilvaPereira** [comment]
@MeisamMulla thanks for reporting, I think the new upcoming schema reader should potentially solve this issue

**Comment by MeisamMulla** [comment]
Any ETA on the release?

**Comment by TiagoSilvaPereira** [comment]
Fixed on 2.0.7
